src/hooks/createSchemaCustomization/index.js

```diff
@@ -37,5 +38,18 @@ module.exports = async (
     cacheDir,
   };
 
+  let activity;
+
+  activity = reporter.activityTimer(`loading DatoCMS schema`, {
+    parentSpan,
+  });
+
+  activity.start();
+
+  await loader.loadSchema();
```

Comment:
@sidharthachatterjee isn't`watchSchema` used somewhere?

Reply:
@stefanoverna You're right. Initially the idea was to use `watchSchema` in `createSchemaCustomization` to add watchers for changes in schema. This would be ideal considering we also planned to remove all loading and watching of schema in `sourceNodes`

However, it looks like `finalizeNodesCreation` depends on `siteId` (which is fetched in the `/site` call) and removing it in `load` broke other things down in the chain. 

Hence, we've kept the current behaviour in `load` and since that already includes watching for schema changes as well, I didn't add another one here.

Reply:
then @sidharthachatterjee shall we get rid of `watchSchema` in the JS client?

Reply:
@matjack1 I figured we should keep it for now but you're right, probably best not to add an API we don't need 😅 I'll remove it for now

Reply:
Thanks!